src/app/chat/view.nim

```diff
@@ -127,10 +127,14 @@ QtObject:
       $(%*(packs))
 
   proc setAvailableStickerPacks*(self: ChatsView, availableStickersJSON: string) {.slot.} =
-    let currAcct = status_wallet.getWalletAccounts()[0] # TODO: make generic
-    let currAddr = parseAddress(currAcct.address)
-    let installedStickerPacks = self.status.stickers.getInstalledStickerPacks()
-    let purchasedStickerPacks = self.status.stickers.getPurchasedStickerPacks(currAddr)
+    let
+      accounts = status_wallet.getWalletAccounts() # TODO: make generic
+      installedStickerPacks = self.status.stickers.getInstalledStickerPacks()
+    var
+      purchasedStickerPacks: seq[int]
+    for account in accounts:
+      let address = parseAddress(account.address)
+      purchasedStickerPacks = self.status.stickers.getPurchasedStickerPacks(address)
```

Comment:
I think you may be overriding the `purchasedStickerPacks` each time you loop, so you only get the last account's packs, no? Should this be a `concat` or nim equivalent?

Reply:
Ah, I see you do the concat inside the function. I guess it's just confusing that `getPurchasedStickerPacks` doesn't actually return the stickers of the address, but of everything that the `StickersModel` has. Maybe it would be worth to rename the function?